SYCL/Plugin/sycl-gpu-cache-config.cpp

```diff
@@ -0,0 +1,39 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t2.out
+// RUN: env SYCL_PI_TRACE=2 %GPU_RUN_PLACEHOLDER %t2.out
+
+// UNSUPPORTED: opencl
```

Comment:
If it is assumed that the test should run on level_zero only it is better to use REQUIRES. Otherwise CUDA will also be valid BE.
```suggestion
// REQUIRES: level_Zero
```